README.md

```diff
@@ -17,6 +17,7 @@ This allows the user to more easily generate type-safe Concourse CI pipelines, t
 * Github List Repos - [`coralogix/eng-concourse-resource-github-list-repos`](https://github.com/coralogix/eng-concourse-resource-github-list-repos)
```

Comment:
do not forget to update the version tags in the README to the new tag you wish to release